.travis.yml

```diff
@@ -1,23 +1,38 @@
 # This config file for Travis CI
 sudo: required
 dist: trusty
+language: generic
+
 services:
   - docker
-cache: ccache
+
+cache:
+  directories:
+    - $HOME/.ccache
+
 notifications:
   email:
     recipients:
       - levi.armstrong@swri.org
+
 env:
   global:
-    - ROS_DISTRO=kinetic
```

Comment:
You could keep this and make the lines in the `matrix` section easier to read.

Compare https://github.com/ros-industrial/industrial_core/blob/7e7d21f5936c4a3a3b9ca666453309aa98507d3b/.travis.yml

Reply:
I added ROS_REPO and NOT_TEST_INSTALL to the global env.